src/main/java/org/prebid/server/proto/openrtb/ext/response/ExtBidPrebid.java

```diff
@@ -2,18 +2,20 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.iab.openrtb.request.Video;
-import lombok.AllArgsConstructor;
-import lombok.Value;
+import lombok.Builder;
+import lombok.Data;
 
 import java.util.Map;
 
 /**
  * Defines the contract for bidresponse.seatbid.bid[i].ext.prebid
  */
-@AllArgsConstructor(staticName = "of")
-@Value
+@Builder(toBuilder = true)
+@Data
```

Comment:
I am not sure if it is good idea to make this mutable.
Just imagine how hard it will be in a future to find state change if anyone can `set` whenever they want. 
@rpanchyk  what do you think?